ui/pages/swaps/index.js

```diff
@@ -266,20 +268,26 @@ export default function Swap() {
     <div className="swaps">
       <div className="swaps__container">
         <div className="swaps__header">
+          <div
+            className="swaps__header-edit"
+            onClick={async () => {
+              await dispatch(navigateBackToBuildQuote(history));
+            }}
```

Comment:
Can we move this to a `useCallback` function?

Reply:
Actually, since these functions are passed into divs, I don't think they are as important.

Reply:
yep usecall back will be overhead here